src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1730,6 +1730,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         <TargetFrameworks>@(_TargetFrameworkInfo)</TargetFrameworks>
         <TargetFrameworkMonikers>@(_TargetFrameworkInfo->'%(TargetFrameworkMonikers)')</TargetFrameworkMonikers>
         <TargetPlatformMonikers>@(_TargetFrameworkInfo->'%(TargetPlatformMonikers)')</TargetPlatformMonikers>
+        <AdditionalPropertiesFromProject>@(_TargetFrameworkInfo->'%(AdditionalPropertiesFromProject)', ';;')</AdditionalPropertiesFromProject>
```

Comment:
What does including `, ';;'` do? My test* didn't look like it did anything.

\* Looked at J's metadata after:
```
<I Include=""initial"">
<M>hi=one;not=two;;hi=three;or=four</M>
</I>
<J Include=""@(I)"">
<ExtraProp>@(I->'%(M)', ';;')</ExtraProp>
<ExtraPropTwo>@(I->'%(M)')</ExtraPropTwo>
</J>
```

Reply:
It specifies what should be used to separate the items in the list when the list is converted to a string: https://docs.microsoft.com/en-us/visualstudio/msbuild/how-to-display-an-item-list-separated-with-commas?view=vs-2019

You would need more than one `I` item when testing it to see the difference.